### DISCUSSION TITLE
Modelado BD

### DISCUSSION THREAD

**Discussion by federico42o** [FrancoGL/ISPC-FullStack#24] [Daily]
Abro este espacio para trabajar sobre el modelado de la base de datos correspondiente al sprint 1

Acá una versión muy simplificada de como podría ser, pueden agregar/quitar cosas si quieren

https://drive.google.com/file/d/1dnvEDTOnlV8dXeZYc5rwxGGmPBDTmyAj/view?usp=sharing

**Comment by ZafkielEdt** [comment]
Le agregué la tabla de pagos, pero como nunca lo he implementado, no estoy seguro de si es viable xD

[Diagrama](https://dbdiagram.io/d/63470c4bf0018a1c5fef28a8)

**Reply by ZafkielEdt** [reply]
@federico42o @cheaguz @Alejo11Lucero @veroivk @javiercorzo37 @pepi1100 @Lukill22 @GabrielSucaria cuál diagrama dejamos?

**Reply by federico42o** [reply]
el tuyo, es algo que tenia pensado lo de la tabla de pagos pero no lo agregue jaja. Aunque creo que haría falta agregarle un campo para el  timestamp

**Reply by veroivk** [reply]
Yo todavía no termino de entender cuantos diagramas están pidiendo. Son 2 o 3?

**Reply by ZafkielEdt** [reply]
> Yo todavía no termino de entender cuantos diagramas están pidiendo. Son 2 o 3?

No específica, pero creo con 1 basta

**Reply by federico42o** [reply]
En una clase de ayer dijeron que no vamos a codear ni meter ningun script relacionado al backend asi que yo creo que con este diagrama ya alcanza para este sprint

**Reply by ZafkielEdt** [reply]
> el tuyo, es algo que tenia pensado lo de la tabla de pagos pero no lo agregue jaja. Aunque creo que haría falta agregarle un campo para el timestamp

Ya agrego ese campo

**Reply by veroivk** [reply]
Bueno.
